src/components/logic/calculate.test.js

```diff
@@ -0,0 +1,67 @@
+// import { render,screen } from "@testing-library/react";
```

Comment:
@Alfred-KInyua and @ChrispaixK great work on your tests, kindly ensure you don't have commented coed in your codebase its not good practice.